README.md

```diff
@@ -18,18 +18,37 @@ The Vantage MCP Server is an open-source tool, written in Golang, that lets you
 
 The Vantage MCP Server currently exposes the following tools, which can be invoked by any compatible MCP client (e.g., Claude, Cursor, Goose):
 
-- `get-costs`
+- `query-costs`
+  - A general purpose way to fetch cost data using VQL.
 
-  - Allows filtering costs by date and VQL.
+- `list-costs`
+
+  - Display all the costs in an associated Cost Report.
 
 - `list-cost-reports`
 
   - List all cost reports available.
 
+- `get-cost-report-forecast`
+
+  - List all Forecasts of spending related to a Cost Report.
+
 - `list-cost-integrations`
 
   - List all cost provider integrations (e.g., AWS, Azure, GCP) available to provide costs data from and their associated accounts.
 
+- `list-cost-providers`
+  - List of just the providers that the given Workspace has shared with it.
+
+- `list-cost-services`
+  - Lists all the Services and their associated Provider that is shared with the given Workspace.
+
+- `list-budgets`
+  - List all Budgets available to compare against a Cost Report and track spending.
+
+- `list-dashboards`
+  - List all dashboards created in the Vantage account.
```

Comment:
Should this be scoped to workspace?

Reply:
The API doesn't filter out based on workspace, it only takes in pagination params.